lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -7,8 +7,16 @@ module Fastlane
 
   module Helper
     class GithubHelper
+      def self.github_token
+        [
+          'GHHELPER_ACCESS', # For historical reasons / backward compatibility
+          'GITHUB_TOKEN',    # Used by the `gh` CLI tool
+        ].map { |key| ENV[key] }
+          .detect { |value| !value.nil? }
```

Comment:
Alternatively:
```suggestion
          .compact.first
```
Which admissibility is O(n) and not O(1) like `detect`/`find` is, but imho more common and acceptable for small datasets like here?